main/src/ca/uwaterloo/flix/language/ast/ops/TypedAstOps.scala

```diff
@@ -169,6 +176,9 @@ object TypedAstOps {
     case Expr.LetRec(sym, _, _, exp1, exp2, _, _, _) =>
       (freeVars(exp1) ++ freeVars(exp2)) - sym
 
+    case Expr.LocalDef(_, sym, fparams, _, _, exp1, exp2, _, _, _) =>
+      (freeVars(exp1) -- fparams.map(_.sym) - sym) ++ freeVars(exp2)
```

Comment:
The minuses get me. Add some extra parens.

Reply:
maybe `let bound = fparams.map(_.sym) + sym`